packages/plugin-snowplow/src/androidMain/kotlin/ly.iterative.itly.snowplow/SnowplowOptions.kt

```diff
@@ -0,0 +1,14 @@
+/**
+ * Android SnowplowPlugin
+ */
+package ly.iterative.itly.snowplow
+
+import android.content.Context
+import com.snowplowanalytics.snowplow.controller.TrackerController
+
+actual open class SnowplowOptions(
+        val androidContext: Context,
+        var vendor: String,
+        var trackerUrl: String?,
```

Comment:
@dantetam `trackerUrl` and `tracker` should default to null to make them optional. Currently in the sample app I need to explicitly set one to null.
```
var tracker: TrackerController? = null
```